src/main/java/ch/wisv/areafiftylan/dto/TransferDTO.java

```diff
@@ -0,0 +1,14 @@
+package ch.wisv.areafiftylan.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * Created by beer on 7-1-16.
+ */
+public class TransferDTO {
```

Comment:
A DTO with just one field can be directly inlined in the Controller method. To do that, you can replace `@RequestBody @Validated TransferDTO transferDTO` with `@RequestBody String goalUsername`. Does essentially the same thing.